### DISCUSSION TITLE
Unknown error in GraphQL Request [DC00:6B85:51800CB:9C8709C:622FE54B]

### DISCUSSION THREAD

**Discussion by mathif92** [community/community#24638] [Apps, API and Webhooks]
<p>Hi!<br>
We’re having this error response when calling the Github GraphQL API:<br>
<code>[{'message': 'Something went wrong while executing your query. Please include </code>DC00:6B85:51800CB:9C8709C:622FE54B<code> when reporting this issue.'}]</code></p>
<p>This error is happening when trying to retrieve the Dependabot alerts, and it doesn’t explain what the issue could be. I’d appreciate any help regarding this.</p>

**Comment by byrneh** [comment]
<p>Hi <a class="mention" href="/mathif92">@mathif92</a><br>
Its hard to test or make suggestion without your actual query that is getting the error.</p>
<p>You could try raising a GitHub support ticket and include that information when reporting it and perhaps the actual query you are using.  They will have access to internal GitHub error information also</p>

**Comment by jsoref** [comment, marked as answer]
<p>Try:<br>
<a href="https://support.github.com/contact?tags=rr-general-technical" class="onebox" target="_blank" rel="noopener">https://support.github.com/contact?tags=rr-general-technical</a></p>
<p>(include at least what you were doing and the value after <code>please include</code> …)</p>

**Comment by Micheal9090** [comment]
<p>Tell me how to create a termux<br>
command</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="Micheal9090" data-post="4" data-topic="238430">
<div class="title">
<div class="quote-controls"></div>
 Micheal9090:</div>
<blockquote>
<p>termux<br>
command</p>
</blockquote>
</aside>
<p>what kind of termux command is that ? and I’m quite not sure if this is related to the post on top</p>
